packages/devtools_app/lib/src/analytics/constants.dart

```diff
@@ -139,6 +139,7 @@ const export = 'export';
 const expandAll = 'expandAll';
 const collapseAll = 'collapseAll';
 const documentationLink = 'documentationLink';
+String topicDocumentationLink(String topic) => '${topic}DocumentationLink';
```

Comment:
It feels strange declaring this in the middle of a list of constants. Can it be put somewhere else?

Reply:
Separated documentation related items. It is good to have them together in spite of different types.